src/SimpleVideoCutter/TaskProcessor.cs

```diff
@@ -189,6 +197,7 @@ public class FFmpegTask
         public FFmpegTaskSelection[]? Selections { get; set; }   
         public long OverallDuration { get; set; }
         public bool Lossless { get; set; }
+        public IActionAfterTaskCompletion? ActionAfterTaskCompletion { get; set; }
```

Comment:
Oh, I never used Shift + E during later development, so I never stumbled upon that. Sorry.

Reply:
This is actually pretty strange, because I think it worked me for the first time I started the app in your branch (i.e. I was able to cut) but then it failed every time. And btw. I did not use `Shift+E`, I just pressed the `Cut` button.